addons/zeus/functions/fnc_canCreateModule.sqf

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+/*
+ * Author: Dystopian
+ * Check whether local player is currently allowed to create a Zeus Module
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * Allowed to create module <BOOL>
+ *
+ * Example:
+ * call ace_zeus_fnc_canCreateModule
+ *
+ * Public: No
+ */
+
+isNil QQGVAR(zeus) && 
```

Comment:
```suggestion
isNil QQGVAR(zeus) && {
```

Reply:
That was what I had at first but I thought readability is better with it on seperate line.